src/pynwb/form/spec/namespace.py

```diff
@@ -256,15 +264,17 @@ def __get_spec_path(cls, ns_path, spec_path):
             return spec_path
         return os.path.join(os.path.dirname(ns_path), spec_path)
 
-    @docval({'name': 'namespace_path', 'type': str, 'doc': 'the path to the file containing the namespaces(s) to load'},
-            {'name': 'resolve', 'type': bool, 'doc': 'whether or not to include objects from included/parent spec objects', 'default': True},
+    @docval({'name': 'namespace_path', 'type': str,
+             'doc': 'the path to the file containing the namespaces(s) to load'},
+            {'name': 'resolve', 'type': bool,
+             'doc': 'whether or not to include objects from included/parent spec objects', 'default': True},
             returns='a dictionary describing the dependencies of loaded namespaces', rtype=dict)
     def load_namespaces(self, **kwargs):
         """Load the namespaces in the given file"""
         namespace_path, resolve = getargs('namespace_path', 'resolve', kwargs)
         # load namespace definition from file
         if not os.path.exists(namespace_path):
-            raise FileNotFoundError("namespace file '%s' not found" % namespace_path)
+            raise OSError("namespace file '%s' not found" % namespace_path)
```

Comment:
See https://github.com/nimiq/promptastic/issues/17. FileNotFoundError is only a Python 3 Exception.